.travis.yml

```diff
@@ -15,8 +15,11 @@ install:
   - sudo apt-get install rethinkdb
   - pip install -e .[test]
   - pip install codecov
+  - pip install flake8
```

Comment:
Since we're at it we may as well update setup.py instead of installing `flake8` here. That is replace `pyflakes` with `flake8`. See https://github.com/bigchaindb/bigchaindb/blob/master/setup.py#L34:

```python
tests_require = [
    'coverage',
    'pep8',
    'pyflakes',    # replace with flake8
    'pylint',
    'pytest',
    'pytest-cov==2.2.1',
    'pytest-xdist',
    'pytest-flask',
]
```